### ISSUE TITLE
插入多条数据的时候指定表名无效（以前的版本没有问题）

### ISSUE THREAD

**Issue by haitgo** [go-xorm/xorm#497]
代码如下
```package main

import (
	"fmt"
	"log"

	_ "github.com/go-sql-driver/mysql"
	"github.com/go-xorm/xorm"
)

type User struct {
	Id   int32  `xorm:"int pk notnull autoincr"  valid:"Int" json:",omitempty"`
	Name string `xorm:"varchar(32)"`
}
type TestUser struct {
	Id   int32  `xorm:"int pk notnull autoincr"  valid:"Int" json:",omitempty"`
	Name string `xorm:"varchar(32)"`
}

var engine *xorm.Engine

func main() {
	var err error
	engine, err = xorm.NewEngine("mysql", "root:hait@/test?charset=utf8")
	if err != nil {
		fmt.Println("err")
	}
	user := &User{Name: "user"}
	_, err = engine.InsertOne(user)
	if err != nil {
		log.Fatal("插入一条数据错误", err)
	}
	user1 := &User{Name: "user1"}
	user2 := &User{Name: "user2"}
	_, err = engine.Insert(user1, user2)
	if err != nil {
		log.Fatal("插入多条数据错误", err)
	}
	user3 := &TestUser{Name: "user3"}
	user4 := &TestUser{Name: "user4"}
	_, err = engine.Table("user").Insert(user3, user4)
	if err != nil {
		log.Fatal("插入多条数据错误", err)
	}

}```
执行结果是：
2016/11/23 14:24:47 插入多条数据错误Error 1146: Table 'test.test_user' doesn't exist
使用2016年5月16日的xorm代码无此错误，运行正常

**Comment by lunny** [comment]
resolved by 64b3df6

**Comment by lunny** [comment]
I will close this issue. Please feel free to reopen it if you have any further information.
